src/main/target/SPRACINGH7EXTREME/target.c

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
This is still missing a Unified Target configuration, see https://github.com/betaflight/betaflight#new-requirements-for-the-submission-of-new-and-updated-targets